docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -52,39 +52,18 @@ The upgrades aim to add features such as liquidity, economic security, usability
 - Gaia v8.0.x
 - Cosmos SDK v0.45
   - Minimal update with small fixes
-- Interchain Security - Required Participation of Provider Chain Validators
-  - The Cosmos solution to shared security that uses IBC Cross Chain Validation (CCV) to relay validator set composition from a Provider Chain (Cosmos Hub) to a Consumer Chain. This validator set is in charge of producing blocks on both networks using separate nodes. Misbehavior on the Consumer Chain results in slashing Provider Chain staking tokens (ATOM).
+- IBC Msg Whitelist to skip MinFee in CheckTX
 - Global Fee Module
   - Allows denoms and min-fees to be governance parameters so gas can be paid in various denoms.
   - Visible on [tgrade](https://github.com/confio/tgrade/tree/main/x/globalfee) already and enabled in [ante.go](https://github.com/confio/tgrade/blob/main/app/ante.go#L72-L92)
 
 ## v9-Lambda Upgrade (expected Q1 2023)
 
 - Gaia v9.0.x
-- Cosmos SDK v0.47
-  - Groups module:
-    - Enables higher-level multisig permissioned accounts, e.g., weight-based voting policies
-  - Gov Module Improvements
-    - Execution of arbitraty transactions instead of just governance proposals.
-    - Enables much more expressive governance module.
-  - Phasing out broadcast mode
-  - Remove proposer based rewards
-  - Consensus-param module
-    - With the deprecation of the param module, this will allow governance or another account to modify these parameters.
-  - Changes required for Interchain Security
-  - Liquid Staking module
-    - Free, secure and low-risk delegations for use in other parts of the Cosmos ecosystem
-    - Features include enabling transfer of rewards and voting rights
-- IBC 5.x
-  - Relayer Incentivisation so that IBC packets contain fees to pay for relayer costs.
-- Interchain Account Message Authorization Module
-  - Authentication module that authorizes any Account to create an Interchain Account on any IBC connected "Host" blockchain that has the Interchain Account IBC module.
-  - Accounts can be private key controlled users, and eventually the Gov Module and any Groups Module.
-- IBC Msg Whitelist to skip MinFee in CheckTX
-- Bech32 Prefix forwarding
-  - <https://github.com/osmosis-labs/bech32-ibc>
-- Liquidity Module Deprecation
-  - Contains forced withdraw of liquidity
+- Cosmos SDK v0.45
```

Comment:
Maybe also link to specific version of the SDK... its a tag of `v45.11`

Reply:
Added link